holesky/README.md

```diff
@@ -1,182 +1,182 @@
-## Installation of EigenDA using docker
+# Holesky
 
-### Prerequisites:
-* Docker installed
-* [EigenLayer CLI](https://github.com/Layr-Labs/eigenlayer-cli) installed
+:::info
+Please ensure you have reviewed the [current Active Operator Set cap](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) and ensure you have sufficient delegated restaked ETH TVL before proceeding.
+:::
 
-> NOTE: For any Docker based commands, if you have installed as root then you might have to append `sudo` in front of the command.
+#### Step 1: Install Prerequisites
 
-### Core operations
-* Register your operator to EigenLayer using [EigenLayer CLI](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/README.md)
+- Complete the EigenLayer CLI installation and registration [here](https://docs.eigenlayer.xyz/operator-guides/operator-installation).
```

Comment:
can we separate this EL registration step? There were operators who missed this step and then failed to opt-in EigenDA and got confused

Reply:
Yes I'll separate these two steps and place emphasis on the registration part.